components/Trade/Advanced/TradingPanel/MarketSelect.tsx

```diff
@@ -152,9 +152,10 @@ export default styled(({ className }: MSProps) => {
                     <SLogo ticker={selectedTracer?.baseTicker ?? 'ETH'} />
                     <div className="my-auto">{selectedTracer?.marketId}</div>
                 </MarketContainer>
-                <div className="ml-auto" onMouseEnter={() => setPopup(true)}>
-                    <MarketSelectDropdownButton arrowUp={popup} />
-                </div>
+                {/*TODO: Add back market dropdown button*/}
+                {/*<div className="ml-auto" onMouseEnter={() => setPopup(true)}>*/}
+                {/*    <MarketSelectDropdownButton arrowUp={popup} />*/}
+                {/*</div>*/}
                 <MarketSelectDropdown
```

Comment:
Wouldnt we remove this as well??

Reply:
Left it there because will need to add it back eventually. Temporary solution.